site/src/sphinx/advanced-spring-webflux-integration.rst

```diff
@@ -0,0 +1,80 @@
+.. _`Spring Boot Reference Guide`: https://docs.spring.io/spring-boot/docs/current/reference/html/
+.. _`armeria-examples`: https://github.com/line/armeria-examples
+
+.. _advanced-spring-webflux-integration:
+
+Using Armeria as a web server of a reactive Spring application
+==============================================================
+
+A user can use Armeria as a web server of his or her reactive Spring application by adding
+`spring-boot-webflux-starter` to the dependencies.
+
+For Maven:
+
+.. code-block:: xml
+
+    <dependency>
+        <groupId>com.linecorp.armeria</groupId>
+        <artifactId>spring-boot-webflux-starter</artifactId>
+    </dependency>
```

Comment:
version?

Reply:
You can insert the real version number instead of `x.y.z`. See https://github.com/line/armeria/blob/master/site/src/sphinx/setup-maven.rst